toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -2038,6 +2045,31 @@ func setGrubCfgSELinux(grubPath string, kernelCommandline configuration.KernelCo
 	return
 }
 
+func setGrubCfgFIPS(bootUUID, grubPath string, kernelCommandline configuration.KernelCommandLine) (err error) {
+	const (
+		enableFIPSPattern = "{{.FIPS}}"
+		enableFIPS        = "fips=1"
+		bootPrefix        = "boot="
+		uuidPrefix        = "UUID="
+	)
+
+	var fips string
+
+	// If EnableFIPS is set, add "fips=1 boot=UUID=<bootUUID value>" to the kernel cmdline in grub.cfg
```

Comment:
I've seen a few places that suggest having `boot=` set when there is NOT a separate boot partition breaks things... I *think* it works fine so long as it does actually point to the partition with boot on it, but I'd like to be sure a system with just a single root partition works ok.

Reply:
Thanks for letting me know. Found some references as well. Will test some behaviors out of curiosity but it is probably safest to add the logic to detect if /boot is a separate partition from the root partition before adding `boot=`.

References:
https://docs.oracle.com/en/operating-systems/oracle-linux/6/security/ol_fips_enable.html
https://www.suse.com/support/kb/doc/?id=000018548